generators/server/templates/src/main/kotlin/package/web/rest/AccountResource.kt.ejs

```diff
@@ -97,34 +95,18 @@ class AccountResource {
     @GetMapping("/account")
     fun getAccount(): UserVM {
         val login = getCurrentUserLogin()
-            .orElseThrow { AccountResourceException::new }
-        val authorities = SecurityContextHolder.getContext().getAuthentication().getAuthorities()
-            .map(GrantedAuthority::getAuthority)
-            .collect(Collectors.toSet());
-        return UserVM(login, authorities);
+            .orElseThrow { AccountResourceException() }
+        val authorities = SecurityContextHolder.getContext().authentication.authorities
+            .mapNotNullTo(mutableSetOf()) { it.authority }
+        return UserVM(login, authorities)
     }
 
-    internal class UserVM {
-        private val login: String;
-        private val authorities: Set<String>;
-
-        @JsonCreator
-        UserVM(login: String, authorities: Set<String>) {
-            this.login = login;
-            this.authorities = authorities;
-        }
-
-        fun isActivated(): boolean {
-            return true;
-        }
+    data class UserVM @JsonCreator constructor(val login: String, val authorities: Set<String>) {
 
-        fun getAuthorities(): Set<String> {
-            return authorities;
+        fun isActivated(): Boolean {
```

Comment:
```suggestion
        fun isActivated() = true
```

Reply:
Nice catch, fixed. Still not used to single expression functions. There are probably several cases within the code base where we can fix these. Will add an issue for that as well